scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Hardcoded Database Credentials

Embedding database credentials directly in the code is a severe security risk[^1]. This can lead to unauthorized database access if the code is compromised.

Recommendation: Use environment variables or a secure configuration management system to store and retrieve database connection strings.
```suggestion
const dbConnection = process.env.DB_CONNECTION_STRING
if (!dbConnection) {
    throw new Error('DB_CONNECTION_STRING environment variable is not set')
}
```